modules/node_modules/@ncigdc/components/Case.js

```diff
@@ -51,9 +51,9 @@ const enhance = compose(
 
 const Case = ({
   node,
-  totalNumCases,
   theme,
   numCasesAggByProject,
+  totalNumCases = Object.keys(numCasesAggByProject).reduce((sum, b) => sum + numCasesAggByProject[b], 0),
```

Comment:
we can always update the code if that happens. I think it is preferable to code to known specs instead of trying to guess the future. it keeps things simple as long as possible